convert_tendl.py

```diff
@@ -118,35 +115,33 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
 # GENERATE HDF5 LIBRARY -- NEUTRON FILES
 
 # Get a list of all ACE files
-neutron_files = glob.glob(release_details[args.release]['neutron_files'])
+neutron_files = release_details[args.release]['neutron_files']
 
 # Create output directory if it doesn't exist
-if not os.path.isdir(args.destination):
-    os.mkdir(args.destination)
+args.destination.mkdir(parents=True, exist_ok=True)
 
```

Comment:
As noted for convert_fendl.py, this modification extends the original behaviour by adding parent directories where required. Just noting this, no action required.